### ISSUE TITLE
Files for scannet dataset evaluation

### ISSUE THREAD

**Issue by Davidyao99** [shiyoung77/OVIR-3D#1]
Great project! I am trying to run your codebase on the scannet 200 dataset. I notice that there is a config.json file, but it does not seem to come with the scannet dataset. How should I generate this config.json file for scannet200 dataset?

Additionally, some files required for evaluating on scannet200 in readme seems to be missing. May I know when will they be released / how do I get access to them?

Thank you so much  ;))

**Comment by shiyoung77** [comment]
Hi David. I pushed some changes to this repo just now, including instructions for installation, a minimal demo, preprocessing [script](https://github.com/shiyoung77/OVIR-3D/blob/main/scannet_related_scripts/scannet_preprocess.py) (which could be used to generate config.json) and some other related files for ScanNet200. Please check the updated readme and let me know if you are able to run the pipeline now. Thanks!

**Comment by Davidyao99** [comment]
Great, thank you so much, everything seems to be working now ;))

Just a side note, but is there a script to visualize the prediction outputs? Perhaps one that shows the pointclouds and their instance segmentations. That will be really useful for debuggin and checking if the algorithm ran successfully ;)) thanks!

**Comment by shiyoung77** [comment]
We do have a visualizer. Please check this [script](https://github.com/shiyoung77/OVIR-3D/blob/main/src/instance_query.py).

**Comment by Davidyao99** [comment]
I am trying to run this [script](https://github.com/shiyoung77/OVIR-3D/blob/main/src/visualize_scannet200_gt.py), but I am getting a 'no module named 'eval'' error. 

I am having issues with downloading the detectron2 package on my local machine. However, I have already generated the predictions in a remote machine, and simply wish to visualize the results. Is this module part of the pycocotools package? or perhaps am I missing something else? Thanks!!

Edit: resolved!

**Comment by Davidyao99** [comment]
I am trying to visualize the groundtruth, but I do not have such a path in my dataset. May I ask how did you generate this "processed data" directory? 
https://github.com/shiyoung77/OVIR-3D/blob/ddc9169d5f5c27a9b28b1befab45e840a5c83218/src/visualize_scannet200_gt.py#L71
Thank you :)))

Edit: found the file!! thanks!

**Comment by Davidyao99** [comment]
Hi, for the scannet extraction script, I notice that you only extract scans where the scene name does not end with "00". This greatly reduces the number of scans available in the validation set. May I ask why this is the case? I assume you used all the videos for your paper?

https://github.com/shiyoung77/OVIR-3D/blob/bcd2e0bafb2a2622a37e4749bb9a61807f0b40a3/scannet_related_scripts/scannet_preprocess.py#L54

Thank you for being so responsive and helpful ;)

**Comment by shiyoung77** [comment]
Oh, this "if" condition should be removed. All 312 scans from the validation set should be used. I'll push a quick fix.

I initially only extracted "00" for quick testing, but later extracted all the others. This "if" condition was used to prevent extracting "00" again.

**Comment by Davidyao99** [comment]
I see, that makes sense. May I ask how did you get this file? [validation.zip](https://github.com/shiyoung77/OVIR-3D/blob/main/scannet_related_scripts/scannet200_instance_gt/validation.zip)

Is it possible for me to get the zip file for the training scenes as well? That will help immensely! Thank you so much!!

**Comment by shiyoung77** [comment]
Please check the preprocessing [script](https://github.com/JonasSchult/Mask3D/blob/main/datasets/preprocessing/scannet_preprocessing.py) from [Mask3D](https://github.com/JonasSchult/Mask3D/). They have instructions in their README.

**Comment by Davidyao99** [comment]
I see thank you!

Are you planning to release the evaluation scripts for scannet200?

**Comment by shiyoung77** [comment]
Yes, likely in a few days.
